src/generate-from-template.js

```diff
@@ -61,7 +61,10 @@ const generateFromTemplate = async ({ templateName }) => {
   log.goToStep(2, templateBanner);
   const templateVersion = getPackageVersion(templateName);
   const storedValues = getStoredValues(templatePackageName, templateVersion);
-  const baseData = await getBaseOptions();
+
+  const flags = templatePackage.templateFlags;
```

Comment:
no need for a variable since `templatePackage.templateFlags` already provides context and it's only used in `handleFlags`